test/e2e_node/system/kernel_validator.go

```diff
@@ -46,6 +47,11 @@ func (k *KernelValidator) Name() string {
 	return "kernel"
 }
 
+var kernelValidatorHelpersByOS = map[string]KernelValidatorHelper{
```

Comment:
probably worth doing the `*_windows.go` here too instead of a function pointer.

Reply:
@brendanburns yup, this was already done in the other PR #53730 made for the e2e_node portions.